assets/component-rte.css

```diff
@@ -1,3 +1,9 @@
+.rte:after {
+  clear: both;
+  content: "";
```

Comment:
For consistency with other references:
```suggestion
  content: '';
```